static/admin/config.yml

```diff
@@ -3,7 +3,7 @@ backend:
   # Swap out "git-gateway" with "test-repo" and 
   # Navigate to localhost:3333/static/admin/index.html to view Netlify CMS locally. 
   name: "git-gateway"
-  branch: master
+  branch: volunteers-to-netlify-cms
```

Comment:
I'll change this back to `master` before I merge, but I wanted us to be able to poke at this page in the Netlify CMS in the deploy preview for now. [Check it out here!](https://deploy-preview-263--texasjusticeinitiative.netlify.com/static/admin/#/collections/pages/entries/aboutUs)

Reply:
Looks really good @nholden . That is too bad about the image previews, I hope they fix that as well!.